uktrade/data-hub-frontend
# Add referral tab into personalise homepage

## Description of change
International trade advisers been instructed to start using referrals, therefore in order to address this requirements referrals tabs being added in their homepage. So they can access the list of referrals using personalised dashboard.

## Test instructions

- From Datahub django admin make it sure personalise dashboard being activated.
- Once being activated, you may able to see an additional `My referrals` tab from the Datahub frontend dashboard.

## Screenshots

### Before

![image](https://github.com/uktrade/data-hub-frontend/assets/28296624/e3987a03-c132-485b-bcec-5eb3f4d8322e)

### After

![image](https://github.com/uktrade/data-hub-frontend/assets/28296624/26179f3f-7256-42e2-8239-6e4c0e84baea)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
